client/src/components/Settings/SettingsViewComponents.res

```diff
@@ -241,4 +241,4 @@ let settingRow = (
 }
 
 let listView = listContent =>
- Html.div(list{Attrs.class(%twc("min-h-[35px] max-h-52 overflow-y-auto overflow-x-hidden w-full bg-grey1 rounded-md"))}, listContent)
+ Html.div(list{Attrs.class(%twc("min-h-[35px] max-h-52 overflow-y-auto overflow-x-hidden w-full bg-grey1 rounded-md canvas-list-scrollbar"))}, listContent)
```

Comment:
We use tailwind-scrollbar (see https://github.com/darklang/dark/blob/main/package.json), which has classes for scrollbar. We use it in the sidebar, so there's probably examples there.

Reply:
Scrollbar width isn't customizable in tailwind-scrollbar. There is `scrollbar` (16 pixels) that is wider than the original, or  `scrollbar-thin` (8 pixels) that is narrower. Do you have a preference?

![scrollbar-width](https://user-images.githubusercontent.com/87861924/202161715-3bf33485-c82d-4600-a867-e3b4d7775df9.png)

Reply:
Whatever is used in the sidebar

Reply:
I believe the sidebar uses the default tailwind scrollbar. I couldn't find any styling for it.